<?xml version="1.0" ?>
<launch>
  <param name="motor_param/forward_P" value="1000" type="double" />
  <param name="motor_param/forward_I" value="0.0" type="double" />
  <param name="motor_param/forward_D" value="0.0" type="double" />
  <param name="motor_param/forward_derivator" value="0.0" type="double" />
  <param name="motor_param/forward_integrator" value="0.0" type="double" />

  <param name="motor_param/angular_P" value="500" type="double" />
  <param name="motor_param/angular_I" value="0.0" type="double" />
  <param name="motor_param/angular_D" value="0.0" type="double" />
  <param name="motor_param/angular_derivator" value="0.0" type="double" />
  <param name="motor_param/angular_integrator" value="0.0" type="double" />

  <param name="motor_param/reverse" value="false" type="bool"/>
 
  <node pkg="turtlebot_teleop" type="turtlebot_teleop_key" name="turtlebot_teleop_keyboard"  output="screen">
  <param name="scale_linear" value="0.5" type="double"/> 
  <param name="scale_angular" value="1.5" type="double"/>
  <remap from="turtlebot_teleop_keyboard/cmd_vel" to="cmd_vel_mux/input/teleop"/>
  </node>

  <node pkg="rosserial_python" type="serial_node.py" name="serial_arduino_node">
    <param name="port" value="/dev/ttyACM0"/>
    <param name="baud" value="57600"/>
  </node>

  <node pkg="robotx_control" type="robotx_control_node" name="motor_node">
  </node>


</launch>
